src/core/create_compilers/RollupCompiler.ts

```diff
@@ -39,10 +39,14 @@ export default class RollupCompiler {
 				this.chunks.push(chunk);
 			},
 			transform: (code: string, id: string) => {
-				if (/\.css$/.test(id)) {
-					this.css_files.push({ id, code });
-					return ``;
+				if (!/\.css$/.test(id)) return;
+				
+				if (/styleInject/.test(code)) {
+					return code;
```

Comment:
What's the difference between `return;` like on the first line and `return code;` that you have here?

Reply:
In this case I'm just mimicking the code that was there previously. That also either returned nothing (if the id did not test positive for `/\.css$/`) or it returned an empty string.